command_line/merge.py

```diff
@@ -106,6 +112,15 @@ def merge_data_to_mtz(params, experiments, reflections):
     ]
     dataset_names = params.output.dataset_names
     crystal_names = params.output.crystal_names
+
+    # check if best_unit_cell is set.
+    best_unit_cell = params.best_unit_cell
+    if best_unit_cell is None:
```

Comment:
Are there any circumstances where we need to explicitly compare this to `None`?
```suggestion
    if not best_unit_cell:
```